client/src/meteor-shim.ts

```diff
@@ -0,0 +1,49 @@
+import axios, { AxiosInstance } from "axios";
+
+export class MeteorShim {
+  _axios!: AxiosInstance;
+
+  startup(cb: () => void) {
+    cb();
+  }
+
+  _subscribe<T = any>(name: string, params: any[], cb: (err: Error | null, data?: T) => void) {
+    this._axios
+      .post(`/subscriptions/${name}`, { arguments: params })
+      .then(response => {
+        cb(null, response.data.return);
+      })
+      .catch(err => cb(err));
+  }
+
+  subscribe(name: string, ...args: any[]) {
+    const cb = args[args.length - 1];
+    const params = args.slice(0, -1);
+
+    this._subscribe(name, params, cb);
+
+    setTimeout(() => this._subscribe(name, params, cb), 1000 * 60);
+  }
+
+  setup() {
+    this._axios = axios.create({
+      baseURL: "http://localhost:8080/",
```

Comment:
This is probably wrong.

Reply:
It's local-only right now. I'll switch it to a REACT_APP_ environment variable :)